apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -47,6 +47,19 @@ const (
 	// annotation is present and status is empty, BMO will reconstruct BMH Status
 	// from the status annotation.
 	StatusAnnotation = "baremetalhost.metal3.io/status"
+
+	// RebootAnnotation is the annotation which tells the host which mode to use
+	// when rebooting - hard/soft
+	RebootAnnotationPrefix = "reboot.metal3.io"
```

Comment:
Could you make sure that duplicate constants do not exist in the controller/provisioner code? E.g. rebootAnnotationPrefix exists in baremetalhost_controller.go. These new constants should be used instead.

Reply:
@dtantsur can we do this part of another PR ?

Reply:
As you wish, just don't forget please.